test_interface/test_main_workflow.py

```diff
@@ -0,0 +1,94 @@
+import os
+import sys
+import unittest
+from PyQt5.QtGui import QKeySequence
+from source.store.project_information_store import project_information_store
+from source.store.project_information_store import wizard_information_store
+from PyQt5.QtCore import Qt
+from PyQt5.QtTest import QTest
+from PyQt5.QtWidgets import *
+from app import Controller
+from PyQt5.QtCore import QStandardPaths as Qs
+
+
+application = QApplication(sys.argv)
+
+
+def key_sequence(widget, sequence):
+    for text in sequence:
+        QTest.keyPress(widget, text)
+
+
+class TestMainWorkflow(unittest.TestCase):
+
+    def setUp(self):
+        self.window = Controller()
+
+    def tearDown(self):
+        os.remove(Qs.writableLocation(Qs.DocumentsLocation) + "/My Title.json")
```

Comment:
What does it do?

Reply:
It's like before all in Junit, so when the test is over, it will delete the generated temp file